src/bin/main.rs

```diff
@@ -0,0 +1,29 @@
+extern crate structopt;
+
+// use fmvparse::reader;
+use fmvparse::mp4::Mp4;
+use std::io;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "fmv-parser",
+    about = "Parse fmv files and extract the internal structures."
+)]
+struct Opt {
+    #[structopt(short, long)]
+    verbose: bool,
```

Comment:
warning: field is never read: `verbose`
  --> src\bin\main.rs:16:5
   |
16 |     verbose: bool,
   |     ^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default